browsers/firefox_android.json

```diff
@@ -54,237 +54,237 @@
         "19": {
           "release_date": "2013-02-19",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/19",
+          "release_notes": "https://www.mozilla.org/firefox/android/19.0beta/releasenotes/",
           "status": "retired"
         },
         "20": {
           "release_date": "2013-04-02",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/20",
+          "release_notes": "https://www.mozilla.org/firefox/android/20.0beta/releasenotes/",
           "status": "retired"
         },
         "21": {
           "release_date": "2013-05-14",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/21",
+          "release_notes": "https://www.mozilla.org/firefox/android/21.0beta/releasenotes/",
           "status": "retired"
         },
         "22": {
           "release_date": "2013-06-25",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/22",
+          "release_notes": "https://www.mozilla.org/firefox/android/22.0beta/releasenotes/",
           "status": "retired"
         },
         "23": {
           "release_date": "2013-08-06",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/23",
+          "release_notes": "https://www.mozilla.org/firefox/android/23.0beta/releasenotes/",
           "status": "retired"
         },
         "24": {
           "release_date": "2013-09-17",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/24",
+          "release_notes": "https://www.mozilla.org/firefox/android/24.0beta/releasenotes/",
           "status": "retired"
         },
         "25": {
           "release_date": "2013-10-29",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/25",
+          "release_notes": "https://www.mozilla.org/firefox/android/25.0beta/releasenotes/",
           "status": "retired"
         },
         "26": {
           "release_date": "2013-12-10",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/26",
+          "release_notes": "https://www.mozilla.org/firefox/android/26.0beta/releasenotes/",
           "status": "retired"
         },
         "27": {
           "release_date": "2014-02-04",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/27",
+          "release_notes": "https://www.mozilla.org/firefox/android/27.0beta/releasenotes/",
           "status": "retired"
         },
         "28": {
           "release_date": "2014-03-18",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/28",
+          "release_notes": "https://www.mozilla.org/firefox/android/28.0beta/releasenotes/",
           "status": "retired"
         },
         "29": {
           "release_date": "2014-04-29",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/29",
+          "release_notes": "https://www.mozilla.org/firefox/android/29.0beta/releasenotes/",
```

Comment:
This shoudn’t point to the beta release notes, but to the [normal release notes](https://www.mozilla.org/firefox/android/29.0/releasenotes/).